nevergrad/optimization/optimizerlib.py

```diff
@@ -1046,6 +1046,9 @@ class SplitOptimizer(base.Optimizer):
         number of optimizers
     num_vars: int or None
         number of variable per optimizer.
+    max_num_vars: int or None
+        maximum number of variables per optimizer. Should not be defined if num_vars is defined:
+        num_vars will be setup internally using max_num_vars.
```

Comment:
```suggestion
        number of variable per optimizer (should not be used if max_num_vars is set)
    max_num_vars: int or None
        maximum number of variables per optimizer. Should not be defined if :code:`num_vars`
         is defined since they will be chosen automatically.
```